### PR TITLE
make schema more consistent with article data

### PR DESCRIPTION
**Pull request by PaulChinkwende**
Our schema and current article data are not completely consistant. This PR makes the schema representative of the article data.

https://app.swaggerhub.com/apis/articles-vivaldi/article/1.1.1#/news-developers/getArticles - Swagger Doc.
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md - OpenAPI 3.0.0 specification.

* [ ] Fixes https://github.com/bbc/simorgh/issues/NUMBER
* [ ] Tests added for new features

* [ ] Test engineer approval

### CODE REVIEW

File: data/schema.yaml [hunk 1]
```diff
@@ -90,14 +88,11 @@ components:
               example: http://www.bbc.co.uk/ontologies/passport/home/News
             articleType:
               type: string
-              enum: [analysis, feature]
+              enum: [analysis, feature, news]
               example: analysis
         seoHeadline:
           type: string
           example: UK pledges extra £44m for Channel border security
-        socialHeadline:
```
Comment (jtart) [inline data/schema.yaml]: Why has this been removed?

File: data/schema.yaml [hunk 2]
```diff
@@ -122,17 +117,17 @@ components:
...
-                  - $ref: '#/components/schemas/Paragraph'
-                  - $ref: '#/components/schemas/Caption'
-                  - $ref: '#/components/schemas/Video'
+                  - $ref: '#/components/schemas/paragraph'
+                  - $ref: '#/components/schemas/caption'
+                  - $ref: '#/components/schemas/video'
                   - $ref: '#/components/schemas/rawVideo'
                   - $ref: '#/components/schemas/rawImage'
-                  - $ref: '#/components/schemas/Model'
+                  - $ref: '#/components/schemas/model'
```
Comment (jtart) [inline data/schema.yaml]: Can `blocks` contain a `model`? That doesn't seem right to me.

File: data/schema.yaml [hunk 3]
```diff
@@ -122,17 +117,17 @@ components:
                   enum: [Headline, Text,altText, paragraph, Image, caption, Video, rawImage, rawVideo]
                 model:
                   oneOf:
-                  - $ref: '#/components/schemas/Article/properties/model/properties/blocks'
-                  - $ref: '#/components/schemas/Text'
-                  - $ref: '#/components/schemas/Image'
-                  - $ref: '#/components/schemas/Headline'
+                  - $ref: '#/components/schemas/article/properties/model/properties/blocks'
```
Comment (jtart) [inline data/schema.yaml]: Can `blocks` contain `blocks`? That doesn't seem right to me.

**Review by pjlee11 — DISMISSED** [review]

**Review by jtart — COMMENTED** [review]
I can't comment because the line hasn't changed in this PR, but the [definition of blocks](https://github.com/bbc/simorgh/blob/bc4b259b5afbaa0189ff58d394e1f805addd73bc/data/schema.yaml#L110-L138) doesn't look right to me.

`blocks` is an array, and its items should be a `block`.

`block` should be an object, and it should be `oneOf` [x items](https://github.com/bbc/simorgh/blob/bc4b259b5afbaa0189ff58d394e1f805addd73bc/data/schema.yaml#L119-L132).

Thoughts @bbc/article-devs-news?

Paul Chinkwende - Your right `blocks` should be an array with the actual block inside it being an object. I think I had trouble showing this in swagger, will try to fix this now.
      .This should be fixed now

File: data/schema.yaml (hunk 1 above)
Comment (PaulChinkwende) [reply data/schema.yaml]: It is not in our article scenarios.

File: data/schema.yaml (hunk 2 above)
Comment (pjlee11) [reply data/schema.yaml]: Agree, blocks shouldn't contain model

File: data/schema.yaml (hunk 3 above)
Comment (pjlee11) [reply data/schema.yaml]: Agree, blocks shouldn't contain blocks

File: data/schema.yaml (hunk 2 above)
Comment (sareh) [reply data/schema.yaml]: I agree that this line should be removed from here.

File: data/schema.yaml [hunk 4]
```diff
@@ -122,17 +117,17 @@ components:
                   enum: [Headline, Text,altText, paragraph, Image, caption, Video, rawImage, rawVideo]
```
Comment (sareh) [inline data/schema.yaml]: `Headline`, `Text`, `Image`, `Video` should be `headline`, `text`, `image`, `video`.

File: data/schema.yaml (hunk 1 above)
Comment (jamesbhobbs) [reply data/schema.yaml]: It's still required, so needs to be added to our scenarios.

Comment (PaulChinkwende) [reply data/schema.yaml]: ok

File: data/schema.yaml (hunk 4 above)
Comment (PaulChinkwende) [reply data/schema.yaml]: Thank you, this change has been made now.

File: data/schema.yaml (hunk 2 above)
Comment (PaulChinkwende) [reply data/schema.yaml]: This was meant to represent the specific instance where model contains just a string inside it and not a block array. I have changed the name of this  to 'modelWithText' to make it more clear.

File: data/schema.yaml (hunk 3 above)
Comment (PaulChinkwende) [reply data/schema.yaml]: Agreed this has been removed.

File: data/schema.yaml (hunk 1 above)
Comment (pjlee11) [reply data/schema.yaml]: Thanks for clarifying. @PaulChinkwende we should add this back in across the schema and I've open the issue https://github.com/bbc/simorgh/issues/262 to get these added to the scenarios.

File: data/schema.yaml [hunk 5]
```diff
@@ -106,42 +105,49 @@ components:
...
-                  - $ref: '#/components/schemas/Paragraph'
-                  - $ref: '#/components/schemas/Caption'
-                  - $ref: '#/components/schemas/Video'
-                  - $ref: '#/components/schemas/rawVideo'
-                  - $ref: '#/components/schemas/rawImage'
-                  - $ref: '#/components/schemas/Model'
-                  - $ref: '#/components/schemas/rawImageBlock'
-                  - $ref: '#/components/schemas/rawVideoBlock'
-
+              type: array
```
Comment (pjlee11) [inline data/schema.yaml]: is there a way to enforce we have to have 1 or more items inside of the blocks array

File: data/schema.yaml [hunk 6]
```diff
@@ -106,42 +105,49 @@ components:
...
           format: date-time
           example: 2017-12-21T17:32:28Z
-        MetaDescription:
+        metaDescription:
           type: boolean
           example: null
         model:
           type: object
           properties:
             blocks:
```
Comment (pjlee11) [inline data/schema.yaml]: This is the same as `model/blocks` nested inside this definition of model. Therefore we should be able to change this to be:
```
    model:
        - $ref: '#/components/schemas/blocks'
```

File: data/schema.yaml [hunk 7]
```diff
@@ -283,6 +301,7 @@ components:
             locator:
               type: string
               example:
```
Comment (pjlee11) [inline data/schema.yaml]: Empty example?

File: data/schema.yaml [hunk 8]
```diff
@@ -106,42 +105,33 @@ components:
           type: string
           format: date-time
           example: 2017-12-21T17:32:28Z
-        MetaDescription:
+        metaDescription:
           type: boolean
           example: null
```
Comment (pjlee11) [inline data/schema.yaml]: A boolean with the example of `null` is wrong, should be true or false. There are a few more example of this, can we change them all please

File: data/schema.yaml (hunk 6 above)
Comment (pjlee11) [reply data/schema.yaml]: Cheers for sorting, github hasn't noticed the diff changed

Comment (PaulChinkwende) [reply data/schema.yaml]: 👍

File: data/schema.yaml (hunk 8 above)
Comment (PaulChinkwende) [reply data/schema.yaml]: Yep

**Review by BogdanDogaru — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]

**Pull request merged**